src/app/_models/targets.ts

```diff
@@ -0,0 +1,37 @@
+import * as am4charts from '@amcharts/amcharts4/charts';
+
+export enum TargetFieldName {
+  THREE_D = 'three_d',
+  HQ = 'hq',
+  TOTAL = 'total'
+}
+
+interface TargetMetaDataBase {
+  value: number;
+  label: string;
```

Comment:
Could you please change this name `label` and `interim`? 
I don't understand what they are supposed to be

Reply:
I've changed `label` to `targetYear` (as per the back end) and `interim` to `isInterim`.

`isInterim` is used in the front-end to determine whether or not the target line on the line chart is dashed or solid.